src/platform/site-wide/user-nav/components/SignInProfileMenu.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import DropDownPanel from '@department-of-veterans-affairs/formation-react/DropDownPanel';
 import IconUser from '@department-of-veterans-affairs/formation-react/IconUser';
 
-import PersonalizationDropdown from './PersonalizationDropdown';
+import { PersonalizationDropdown } from './PersonalizationDropdown';
```

Comment:
Pretty sure you want to keep this as the connected version. It's only with unit tests where you want to bring in the Redux-free version of the component.

Reply:
ah, thank you for catching that